lib/Notification/Notifier.php

```diff
@@ -186,18 +186,19 @@ public function handleImportDone(INotification $notification, string $languageCo
 		$param = $notification->getSubjectParameters();
 
 		$sourceUser = $this->getUser($param['sourceUser']);
-		$path = $this->$param['path'];
+		$targetUser = $this->getUser($param['targetUser']);
+		$path = $param['path'];
 		$importFile = $this->getImportFile($sourceUser, $path);
 
 		$notification->setRichSubject($l->t('User import done'))
 			->setParsedSubject($l->t('User import done'))
 			->setRichMessage(
-				$l->t('Your import of {user} has completed: {file}'),
+				$l->t('Your import to {user} has completed: {file}'),
```

Comment:
Import should not link to imported file, or at least not with this wording, as the file is not a result of the import.
Eventually «Your import of {file} into {user} has completed.».

Reply:
I think it's ok to link to the file because the user will likely want to delete it after a successful import.

@come-nc I prefer your wording, please adjust